test/session_spec.lua

```diff
@@ -168,6 +169,23 @@ test_session("Session using ChidProcessStream", function ()
   return Session.new(proc_stream)
 end)
 
+describe("ChildProcessStream", function()
+  it("can capture exit code", function()
+    local proc_stream = ChildProcessStream.spawn({
+      nvim_prog, '-u', 'NONE', '--embed',
```

Comment:
is there a reason `--clean` is not used? 

```suggestion
      nvim_prog, '--clean', '--embed',
```

Reply:
I copy-pasted the argument in the test above, I'll do that.